File: app/src/main/java/ai/elimu/content_provider/rest/StoryBooksService.java
package ai.elimu.content_provider.rest;

import java.util.List;

import ai.elimu.model.gson.v2.content.StoryBookGson;
import retrofit2.Call;
import retrofit2.http.GET;

public interface StoryBooksService {

    @GET("content/storybooks")
    Call<List<StoryBookGson>> listStoryBooks();
}

File: app/src/main/java/ai/elimu/content_provider/rest/StoryBooksService.kt
package ai.elimu.content_provider.rest

import ai.elimu.model.v2.gson.content.StoryBookGson
import retrofit2.Call
import retrofit2.http.GET

interface StoryBooksService {
    @GET("content/storybooks")
    fun listStoryBooks(): Call<List<StoryBookGson>>
}
